io.github.hurricangame.hurrican.yml

```diff
@@ -0,0 +1,45 @@
+app-id: io.github.hurricangame.hurrican
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: hurrican
+rename-desktop-file: Hurrican.desktop
+finish-args:
+  - --device=all
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+modules:
+  - name: libopenmpt
+    buildsystem: autotools
+    config-opts:
+      - --without-mpg123
+      - --without-portaudio
+      - --without-portaudiocpp
+      - --without-ogg
+      - --without-vorbis
+      - --without-vorbisfile
+      - --without-sndfile
+      - --without-flac
+    cleanup:
+      - /include
+    sources:
+      - type: archive
+        url: https://lib.openmpt.org/files/libopenmpt/src/libopenmpt-0.6.2+release.autotools.tar.gz
+        sha256: 50c0d62ff2d9afefa36cce9f29042cb1fb8d4f0b386b81a0fc7734f35e21e6b6
+  - name: hurrican
+    buildsystem: cmake-ninja
+    builddir: true
+    subdir: "Hurrican"
+    config-opts:
+      - "-DOPENMPT=ON"
+      - "-DCMAKE_BUILD_TYPE=Release"
+    post-install:
+      - install -Dm644 ../../Hurrican.desktop /app/share/applications/Hurrican.desktop
```

Comment:
you should also install the .appdata.xml

(it should also be submitted upstream)

this is why the build failed

Reply:
Fixed. I don't think that caused an error when building locally, is that normal?